### PR TITLE
simpar Validation

### PR DESCRIPTION
**Pull request by rymarinelli**
https://metworx.atlassian.net/browse/TDT-146

### CODE REVIEW

**Comment by seth127** [comment]
This looks great, and the docs that were created from it are looking good. I think my only comment is to update the `NEWS.md` file for the new release. I think it should look like this:

```
# simpar 0.1.1

- This release adds documentation for validation purposes, but makes no functional changes.

# simpar 0.1.0

- The minimum number of degrees of freedom when simulating OMEGA or SIGMA
  is equal to the number of rows in the matrix rather than the length (#6)

- Add a `NEWS.md` file to track changes to the package
```

File: inst/validation/simpar-stories.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+SIM-S001:
+  name: create parameters for simulation
+  description: As a user, I would  to create parameters for simulation with uncertainty.
```
Comment (kylebaron) [inline inst/validation/simpar-stories.yaml:3]:
```suggestion
  description: As a user, I want to create parameters for simulation with uncertainty.
```

File: inst/validation/simpar-requirements.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  tests:
+  - SMP-SIM-002
+
+SIM-R003:
+  description: return omega matrix as expected
+  tests:
+  - SMP-SIM-003
+
+SIM-R004:
+  description: produces simulations consistently
```
Comment (kylebaron) [inline inst/validation/simpar-requirements.yaml:17]:
```suggestion
  description: simulations are reproducible when seed is set
```

File: inst/validation/simpar-requirements.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  tests:
+  - SMP-SIM-009
+
+SIM-R010:
+  description: confirms minimum degrees of freedom is equal number of rows per matrix
+  tests:
+  - SMP-SIM-010
+
+SIM-R011:
+  description: confirms consistent number of signficant figures
```
Comment (kylebaron) [inline inst/validation/simpar-requirements.yaml:52]:
```suggestion
  description: confirms expected number of signficant figures
```

**Review by kylebaron — APPROVED** [review]

**Comment by kylebaron** [comment]
LGTM; a couple of suggestions on wording. Thanks for punching this out, @rymarinelli .

**Pull request merged**